style.css

```diff
@@ -0,0 +1,269 @@
+* {
+	box-sizing:	border-box;
+	font-family: "Arial";
+	font-size: 15px;
+	background-color:#000000;
+	letter-spacing: 2px;
+}
+/*TAGS*/
+nav {
+	position: fixed;
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	background-color: rgba(0,0,0,1.0);
+	width: 100%;
+	margin-left: 0;
+	top: 0;
+	left: 0;
+	z-index: 1;
+	
+	
+}
+main {
+	display: flex;
+	flex-wrap: nowrap;
+	flex-direction: row;
+	gap: 10px;
+	margin-top: 100px;
+
+}
+
+/*CLASSES*/
+/*NAVBAR BTNS*/
+.nav-btns-left {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: flex-start;
+	gap: 15px;
+	margin-left: 20px;
+	margin-top:10px;
+	color: #000000;
```

Comment:
this colour here is wrong, it would mean black text on a black background (it also doesn't work)